pecl-compat/compat.h

```diff
@@ -77,9 +77,13 @@
 #endif
 
 #ifdef PHP_WIN32
-#include <win32/php_stdint.h>
+#	if PHP_MAJOR_VERSION < 8
+#		include <win32/php_stdint.h>
+#	else
+#		include <inttypes.h>
+#	endif
 #else
-#include <inttypes.h>
+#	include <inttypes.h>
 #endif
```

Comment:
I think this can be simplified:
````.c
#if defined(PHP_WIN32) && PHP_MAJOR_VERSION < 8
# include <win32/php_stdint.h>
#else
# include <inttypes.h>
#endif
````

Reply:
Good point